private_sharing/templates/email/notify-withdrawal.html

```diff
@@ -0,0 +1,32 @@
+{% load private_sharing %}
+
+<p>
+  Hello,
+</p>
+
+<p>
+  This email is to inform you that a member, {{ project.project_member.project_member_id }},
+  has withdrawn their participation from your project.
+</p>
+<p>
+  For your convenience, we maintain a list of project members that have requested
+  withdrawal.  It can be found at {{ withdrawn_url }}.
+</p>
+<p>
+  {% erasure_requested_txt erasure_requested %}
```

Comment:
I think I'd prefer an `{% if erasure_requested %}` block that only shows up if it's true. I'm thinking about projects that don't offer erasure in the first place - it's weird to get a statement like "erasure not requested" when it's never possibly true? Feels like it's mainly important to include when it's true, nothing needs to be said otherwise.

Reply:
Okay, I'll knock that out right quick.